NEWS.md

```diff
@@ -1,3 +1,14 @@
+# cfbscrapR 1.0.2
+- Fix issues with win probability added
+- Remove the `add betting columns()` and the current betting win probability model from the `cfb_pbp_data()` function
+- Fix downs turnovers and several other discrepancies in EPA computations
```

Comment:
Is it possible to add the plot that Nate Manzo shared? If we can do a before/after I think itwould help here. It's a markdown so it should be possible to attach.

Reply:
![EPA_YardsGained_cfbscrapR](https://user-images.githubusercontent.com/25425933/92576938-187b5080-f258-11ea-8149-075c6140e024.png)
![EPA_YardsGained_cfbscrapR2](https://user-images.githubusercontent.com/25425933/92576940-1913e700-f258-11ea-8504-c22a672600b3.png)
![new_version](https://user-images.githubusercontent.com/25425933/92576943-19ac7d80-f258-11ea-8dfc-3f20d070c082.png)
![test](https://user-images.githubusercontent.com/25425933/92576944-19ac7d80-f258-11ea-93bb-c62cf2fa2b53.png)
![WPA_YardsGained_cfbscrapR](https://user-images.githubusercontent.com/25425933/92576946-19ac7d80-f258-11ea-8f9b-cd6d895a8cb1.png)
![WPA_YardsGained_cfbscrapR2](https://user-images.githubusercontent.com/25425933/92576948-1a451400-f258-11ea-9d5f-fa4b6295fce0.png)